storybook/__snapshots__/storyshots.test.js.snap

```diff
@@ -38949,7 +38949,7 @@ exports[`Storyshots FeeLabelToggle default 1`] = `
               ]
             }
           >
-            tokenValue
+            £NaN
```

Comment:
Pls handle these `NaN` outputs by providing the correct redux `rates` state in `FeeLabelToggle.stories.js`, in order for snapshot test to calculate some valid (non-`NaN` value).

Reply:
It seems that the reason for NaN value is that under the tests `formatTransactionFee(txFeeInWei, gasToken)` in FeeLabelToggle.js returns `'tokenValue'` string, which later causes `parseFloat(feeDisplayValue)` to return `NaN`. 

Looking into details it seems that the culprit is that we try to parse user visible string output from `formatTransactionFee`. This parsing is used because we need to go from transaction amount in Wei to amount in ETH. Simplies workaround would be to get the ETH amount by using something like:

```
const feeValue = getFormattedTransactionFeeValue(txFeeInWei, gasToken);
...

const feeInFiat = parseFloat(feeValue) * getRate(rates, gasTokenSymbol, fiatCurrency);
```
While keeping the `feeDisplayValue` for UI display purposes only.

Reply:
Thanks. It worked.

Reply:
I have pushed the changes. I am no longer getting NaN. However my 2 test cases from authActions.test.js are failing even in develop branch where no changes have been done. I am assuming this is a machine specific issue. Hence I have skipped for now.